specification/mediaservices/resource-manager/Microsoft.Media/preview/2018-03-30-preview/StreamingPoliciesAndStreamingLocators.json

```diff
@@ -0,0 +1,1248 @@
+          "type": "string",
+          "description": "LicenseAcquistionUrlTemplate is used to point to user speicified service to delivery content keys"
+        }
+      },
+      "type": "object",
+      "description": "Class for EnvelopeEncryption encryption scheme"
+    },
+    "CommonEncryptionCenc": {
+      "properties": {
+        "enabledProtocols": {
+          "$ref": "#/definitions/EnabledProtocols",
+          "description": "Representing supported protocols"
+        },
+        "clearTracks": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/TrackSelection"
+          },
+          "description": "Representing which tracks should not be encrypted"
+        },
+        "contentKeys": {
+          "$ref": "#/definitions/StreamingPolicyContentKeys",
+          "description": "Representing default content key for each encryption scheme and separate content keys for specific tracks"
+        },
+        "drm": {
+          "$ref": "#/definitions/CencDrmConfiguration",
+          "description": "Configuration of DRMs for CommonEncryptionCenc encryption scheme"
+        }
+      },
+      "type": "object",
+      "description": "Class for envelope encryption scheme"
+    },
+    "CommonEncryptionCbcs": {
+      "properties": {
+        "enabledProtocols": {
+          "$ref": "#/definitions/EnabledProtocols",
+          "description": "Representing supported protocols"
+        },
+        "clearTracks": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/TrackSelection"
+          },
+          "description": "Representing which tracks should not be encrypted"
+        },
+        "contentKeys": {
+          "$ref": "#/definitions/StreamingPolicyContentKeys",
+          "description": "Representing default content key for each encryption scheme and separate content keys for specific tracks"
+        },
+        "drm": {
+          "$ref": "#/definitions/CbcsDrmConfiguration",
+          "description": "Configuration of DRMs for current encryption scheme"
+        }
+      },
+      "type": "object",
+      "description": "Class for CommonEncryptionCbcs encryption scheme"
+    },
+    "StreamingPolicyProperties": {
+      "properties": {
+        "created": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Creation time of Streaming Policy",
+          "readOnly": true,
+          "x-nullable": false
+        },
+        "defaultContentKeyPolicyName": {
+          "type": "string",
+          "description": "Default ContentKey used by current Streaming Policy"
+        },
+        "envelopeEncryption": {
+          "$ref": "#/definitions/EnvelopeEncryption",
+          "description": "Configuration of EnvelopeEncryption"
+        },
+        "commonEncryptionCenc": {
+          "$ref": "#/definitions/CommonEncryptionCenc",
+          "description": "Configuration of CommonEncryptionCenc"
```

Comment:
>CommonEncryptionCenc [](start = 43, length = 20)

Description is not the best here :(

Reply:
Agreed.  I updated these and we will do a wider review to ensure we have more helpful descriptions.